Dungeons and Dragons Tool/dndtool/lib/dndtool/player_cache_server.ex

```diff
@@ -0,0 +1,35 @@
+defmodule DnDTool.Player_Cache_Server do
+
+  use GenServer
+
+  alias DnDTool.Player_Cache, as: Player_Cache
+
+  @player_cache :player_cache
+
+  def start_link() do
+    GenServer.start_link(__MODULE__, [], name: __MODULE__)
+  end
+
+  ############################
+  # GenServer Implementation #
+  ############################
+
+  def init(state) do
+    Agent.start_link(&Player_Cache.load_players/0, name: @player_cache)
+    {:ok, state}
+  end
+
```

Comment:
Here you're running a server than then talks to another server. 

Perhaps instead you might simply to the `Agent.start_link` in this module's `start_link` function, and then manipulate the agent server directly? (Same for the spell cache)